bpf/mb_bind.bpf.c

```diff
@@ -17,6 +17,9 @@ limitations under the License.
 #include "headers/helpers.h"
 #include "headers/mesh.h"
 
+const volatile short unsigned int out_redirect_port = 15001;
+const volatile unsigned int sidecar_user_id = 1337;
```

Comment:
Should we change it to default kuma user id or we good with this? `5678`

Reply:
let's stick as close as possible to the merbridge defaults